src/main/java/by/custom/utilcalculator/service/CustomSendMessage.java

```diff
@@ -0,0 +1,11 @@
+package by.custom.utilcalculator.service;
+
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+
+public class CustomSendMessage extends SendMessage {
+
+    public CustomSendMessage(final String chatID, final String answer) {
+        super.setChatId(chatID);
```

Comment:
maybe there is constructor `super(chatID, answer)`?

Reply:
Yes, this construction really was there and I could use it without a custom wrapper. Thank you for advice, fixed